docs/unique/lisp-compiler-integration.md

```
@@ -1193,178 +1193,178 @@ console.log(js);
 <!-- GENERATED-SECTIONS:DO-NOT-EDIT-BELOW -->
 ## Related content
-- [Interop and Source Maps](interop-and-source-maps.md)
-- [Lispy Macros with syntax-rules](lispy-macros-with-syntax-rules.md)
-- [compiler-kit-foundations](compiler-kit-foundations.md)
-- [set-assignment-in-lisp-ast](set-assignment-in-lisp-ast.md)
-- [zero-copy-snapshots-and-workers](zero-copy-snapshots-and-workers.md)
-- [Language-Agnostic Mirror System](language-agnostic-mirror-system.md)
-- [Polyglot S-expr Bridge: Python-JS-Lisp Interop](polyglot-s-expr-bridge-python-js-lisp-interop.md)
-- [Promethean Agent Config DSL](promethean-agent-config-dsl.md)
-- [template-based-compilation](template-based-compilation.md)
+- [[docs/unique/interop-and-source-maps|Interop and Source Maps]]
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)